app/views/proposals/index.html.haml

```diff
@@ -29,6 +29,8 @@
               .event-meta.margin-top
                 %span.event-meta
                   = link_to 'View Guidelines', event_path(event.slug)
+          %span.event-meta
```

Comment:
Move this into the event-info-block.  Ideally, I'd see it just below the CFP status area.  Also, add a check that only shows the submit button if the cfp is open.